test/network_test.rb

```diff
@@ -0,0 +1,38 @@
+require 'minitest/autorun'
+require 'minitest/pride'
+require './lib/character'
+require './lib/show'
+require './lib/network'
+
+class NetworkTest < Minitest::Test
+
+  def setup
+    @kitt = Character.new({name: "KITT", actor: "William Daniels", salary: 1_000_000})
+    @michael_knight = Character.new({name: "Michael Knight", actor: "David Hasselhoff", salary: 1_600_000})
+    @knight_rider = Show.new("Knight Rider", "Glen Larson", [@michael_knight, @kitt])
+    @leslie_knope = Character.new({name: "Leslie Knope", actor: "Amy Poehler", salary: 2_000_000})
+    @ron_swanson = Character.new({name: "Ron Swanson", actor: "Nick Offerman", salary: 1_400_000})
+    @parks_and_rec = Show.new("Parks and Recreation", "Michael Shur & Greg Daniels", [@leslie_knope, @ron_swanson])
+    @nbc = Network.new("NBC")
+  end
+
+  def test_it_exists
+    assert_instance_of Network, @nbc
+  end
+
+  def test_network_attributes
+    assert_equal "NBC", @nbc.name
+    assert_equal [], @nbc.shows
```

Comment:
this would be a good place to stop this attributes test, and the add_show method calls would be in a separate test that checks the `highest_paid_actor` method, and then making one more test with the same add_show calls to test the `payroll` method